tests/lib.rs

```diff
@@ -172,4 +173,26 @@ mod tests
         assert_eq!(population.individual(0, GAPopulationSortBasis::Scaled).fitness(), i_f_m);
         assert_eq!(population.individual(1, GAPopulationSortBasis::Scaled).fitness(), i_f);
     }
+
+    #[test]
+    #[allow(unused_variables)]
+    fn test_rank_selector()
+    {
```

Comment:
Let's add a call to ga_test_setup() here.. and also to test_sort_population() (forgot about it)